go/apps/simpleexample/SimpleDomain/copybins

```diff
@@ -1,9 +1,9 @@
 #
-
```

Comment:
I'd rather keep all the calls and defines explicit for clarity.  I'd be happy to do this in addition, however.  Note that this is a little weird for C++  functions though.